src/containers/PostList/components/PostItem/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import './style.css';
+import like from '../../../../images/like.png';
+
+// 无状态组件
+function PostItem(props) {
+    const { post } = props;
+    return (
+        <li className="postItem">
+            <div className="title">{post.title}</div>
+            <div>
+                创建人： <span>{'frank'}</span>
```

Comment:
Not completed PostItem, like the user here, currently it's hard coded. Once user parts completed, can update the code here.